### PR TITLE
coffee transcriptome news

### PR DESCRIPTION
**Pull request by Deeptivarshney**
Hi @teresa-m, 

Feel free to modify the content :)

### CODE REVIEW

File: content/news/2024-05-02_Coffee/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Transcriptomic Data Analysis Using the Galaxy Platform"
+date: "2024-05-02"
+authors: Deepti Varshney
+tease: "Book Chapter: Transcriptomic Data Analysis Using the Galaxy Platform: Coffee (Coffea arabica L.) Flowers as Example"
```
Comment (teresa-m) [inline content/news/2024-05-02_Coffee/index.md:5]:
```suggestion
tease: "An interesting book chapter presents an overview of a transcriptomics data analysis pipeline using Galaxy, specifically designed for studying gene expression in coffee flowers. This pipeline can be leveraged in genetic improvement programs, for instance, to combat climate change "
tags: [paper]

**Review by teresa-m — APPROVED** [review]
Thanks a lot Deepti.
I added a suggestion for the teaser.

File: content/news/2024-05-02_Coffee/index.md (hunk 1 above)
Comment (Deeptivarshney) [reply content/news/2024-05-02_Coffee/index.md]: Thanks :)

**Comment by bgruening** [comment]
Thanks :-)

**Pull request merged**